### PR TITLE
docs: change generator scripts

### PR DESCRIPTION
**Pull request by thepassle**

### CODE REVIEW

**Comment by thepassle** [comment]
@daKmoR Thinking how to split up the generators with the cli menu, what do you think of:

- App generators
  - starter app
  - barebones app
  - etc
- Scaffold generators
  - vanilla
  - building
  - demoing
  - etc
- Upgrade generators
  - linting
  - building
  - testing
  - etc
- Sub generators
  - linting-eslint
  - linting-prettier
  - testing-karma
  - testing-wallaby
  - etc-etc

**Comment by daKmoR** [comment]
there are no more scaffold generator... it's just a question in the upgrade generators.
e.g. `do you want to have some example tests as well?`

**Comment by thepassle** [comment]
Ah thats nice, so it would be:

- App generators
  - starter app
  - barebones app
  - etc
- Upgrade generators
  - linting
  - building
  - testing
  - etc
- Sub generators
  - linting-eslint
  - linting-prettier
  - testing-karma
  - testing-wallaby
  - etc-etc

then?

File: packages/generator-open-wc/README.md [hunk 1]
```diff
@@ -59,7 +47,7 @@ yo open-wc:scaffold-vanilla
 
 ### Available scaffold generators:
 
-- `yo open-wc:scaffold-vanilla`<br/> 
+- `scaffold-vanilla`<br/>
```
Comment (daKmoR) [inline packages/generator-open-wc/README.md]: is gone... "replaced" with `wc-lit-element` but it only scaffolds webcomponent and does not install everything... 
it seem the more prefered way of working is to make a small thing (without much) play around and once you feel ready you add testing/demoing/building/...

File: packages/prettier-config/README.md [hunk 2]
```diff
@@ -10,10 +10,10 @@ This is part of the default [open-wc](https://open-wc.org/) recommendation
 
 ## Setup
 ```bash
-npm i -g yo
-npm i -g generator-open-wc
-
-yo open-wc:linting-prettier
+npm init open-wc
```
Comment (daKmoR) [inline packages/prettier-config/README.md]: not sure if we wanna offer this deep of a menu... i think it's too confusing... if you want to run only this specific generator you can only do it via providing the actual generator name for it e.g. `npm init open-wc linting-prettier`

Comment (thepassle) [reply packages/prettier-config/README.md]: yeah, i'll clean this up tomorrow, it was more of a draft rn :)

**Comment by thepassle** [comment]
@daKmoR PTAL

I removed the 'scaffold generators' as they were, since they're no longer there, and kind of 'merged' the application generators with the scaffold generators.

So to clarify, the scaffold generators are now:
- starter-app
- wc-lit-element

**Comment by thepassle** [comment]
I undid the symlink for `packages/generator-open-wc` and the file in the `docs` folder, and added the prefix: `npm init @open-wc`

**Pull request merged**